docs/pt/docs/help-fastapi.md

```diff
@@ -0,0 +1,106 @@
+# Ajuda FastAPI - Obtendo Ajuda
+
+Você gosta do **FastAPI**?
+
+Gostaria de ajudar o FastAPI, outros usuários, e o autor?
+
+Ou gostaria de obter ajuda para utilizar o **FastAPI**?
+
+Há maneiras muito simples de ajudar (algumas envolvem somente um ou dois cliques). 
+
+Há várias maneiras de buscar ajuda também.
+
+## Favorite **FastAPI** no Github
+
+Você pode "_star_" o FastAPI no GitHub (clicando no botão com a estrela no canto superior direito): <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>.
+
+Ao favoritar o repositório, outros usuários serão capazes de encontrá-lo mais facilmente e ver que o framework foi útil à outros usuários.
+
+## Fique de olho nos lançamentos de novos recursos
+
+Você pode "_watch_" o FastAPI no GitHub (clicando no botão escrito 'Watch' no canto superior direito): <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>.
+
+Depois de clicar o botão é só selecionar _"Releases only"_
+
+Fazendo isso, você vai receber notificações (no seu e-mail) todas as vezes que tiver um novo lançamento (uma nova versão) do **FastAPI** com correções de _bugs_ e novos recursos.
+
+## Entre no _chat_
```

Comment:
This section has been removed on the original documentation. The Gitter chat is no longer recommended...